### PR TITLE
Add support for system webviews

### PR DESCRIPTION
**Pull request by kpanwar**
This PR includes support for SafariViewController in iOS and Chromtabs/Chrome browser in Android. There is no fallback support in iOS as we will support iOS 9 minimum.

### CODE REVIEW

File: samples/AndroidTestApp/MainActivity.cs [hunk 1]
```diff
@@ -60,10 +60,10 @@ protected override void OnCreate(Bundle bundle)
             this.accessTokenTextView = FindViewById<TextView>(Resource.Id.accessTokenTextView);
 
             sts.Authority = "https://login.microsoftonline.com/common";
-            sts.ValidClientId = "b92e0ba5-f86e-4411-8e18-6b5f928d968a";
-            sts.ValidScope = new [] { "https://msdevex-my.sharepoint.com"};
+            sts.ValidClientId = "5a434691-ccb2-4fd1-b97b-b64bcfbc03fc";
```
Comment (weijjia) [inline samples/AndroidTestApp/MainActivity.cs]: Are we going to scrub those info before merging or is it ok to check them in?

File: samples/AndroidTestApp/Properties/AndroidManifest.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="AndroidTestApp.AndroidTestApp" android:versionCode="1" android:versionName="1.0" android:installLocation="auto">
-	<uses-sdk android:maxSdkVersion="12" />
+	<uses-sdk android:maxSdkVersion="12" android:minSdkVersion="19" />
```
Comment (weijjia) [inline samples/AndroidTestApp/Properties/AndroidManifest.xml]: max sdk 12???? and min 19??

Comment (kpanwar) [inline samples/AndroidTestApp/Properties/AndroidManifest.xml]: fixed.

File: samples/AndroidTestApp/MainActivity.cs (hunk 1 above)
Comment (kpanwar) [inline samples/AndroidTestApp/MainActivity.cs]: fixed across all sample apps

**Comment by weijjia** [comment]
:shipit: 🚀 🐑 🚢

**Pull request merged**